tests/libmonitoring/scaling.go

```diff
@@ -39,30 +38,37 @@ func (s *Scaling) BackupScale(operatorName string) {
 	Eventually(func() error {
 		virtOperatorCurrentScale, err := s.virtClient.AppsV1().Deployments(flags.KubeVirtInstallNamespace).GetScale(context.TODO(), operatorName, metav1.GetOptions{})
 		if err == nil {
-			s.scales[operatorName] = virtOperatorCurrentScale
+			s.scales[operatorName] = virtOperatorCurrentScale.Spec.Replicas
 		}
 		return err
 	}, 30*time.Second, 1*time.Second).Should(BeNil())
 }
 
+func (s *Scaling) GetScale(operatorName string) int32 {
+	return s.scales[operatorName]
+}
+
 func (s *Scaling) UpdateScale(operatorName string, replicas int32) {
 	By(fmt.Sprintf("Updating scale for %s to %d", operatorName, replicas))
-	scale := s.scales[operatorName].DeepCopy()
-	scale.Spec.Replicas = replicas
 
 	Eventually(func() error {
-		_, err := s.virtClient.
-			AppsV1().
-			Deployments(flags.KubeVirtInstallNamespace).
-			UpdateScale(context.TODO(), operatorName, scale, metav1.UpdateOptions{})
+		scale, err := s.virtClient.AppsV1().Deployments(flags.KubeVirtInstallNamespace).GetScale(context.TODO(), operatorName, metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+
+		scale.Spec.Replicas = replicas
+		scale.ResourceVersion = ""
```

Comment:
What's that supposed to do? Can you use a Patch instead?

Reply:
we should updatescale and not patch here
but this can be removed